index.Rmd

```diff
@@ -8,6 +8,9 @@ biblio-style: apalike
 link-citations: yes
 description: "Description about Course/Book."
 favicon: assets/dasl_favicon.ico
+output:
+    bookdown::word_document2:
+      toc: true
```

Comment:
Is this necessary in addition to the `output_format = 'all'`? Sometimes bookdown has things that are redundant and confusing. So just wondering.

Reply:
Locally, `bookdown::render_book(output_format = "all")` renders the docx but `bookdown::render_book(output_format = "all")` doesn't. I agree it's weird and kind of redundant.

Reply:
From the documentation:

> If using NULL then the output format is the first one defined in the YAML frontmatter in the input file (this defaults to HTML if no format is specified there).

Reply:
I'm always so confused by the various places where yamls are specified and how to know which is prioritized. But if you have it figured out then sounds good to me! :+1:

Reply:
haha me too - it took me forever to find that bit in the docs :P